framework/packages/abstract-standalone/src/endpoints/instantiate.rs

```diff
@@ -0,0 +1,59 @@
+use abstract_sdk::{
+    feature_objects::{AnsHost, VersionControlContract},
+    AbstractSdkResult,
+};
+use abstract_std::{
+    objects::module_version::set_module_data,
+    standalone::{BaseInstantiateMsg, StandaloneState},
+    AbstractError,
+};
+use cosmwasm_std::{Addr, DepsMut, Env};
+use cw2::set_contract_version;
+
+use crate::state::StandaloneContract;
+
+impl StandaloneContract {
+    /// Call this method on instantiating of the standalone
+    pub fn instantiate(
```

Comment:
So this means that users can't send a custom instantiate Msg to the Standalone (or migrate msg later ?)

Reply:
Using those methods is optional and if you want to use it you can also have any structure, even without base standalone instantiate message(for example having vc and ans addrs included in your message), as long as you sent both here!

Reply:
> So this means that users can't send a custom instantiate Msg to the Standalone (or migrate msg later ?)

We don't implement any endpoints for the developer so it's completely optional.

Reply:
Oh got it thanks